client/app/main/student/partial/outreaches.html

```diff
@@ -39,8 +39,8 @@
         <dt>Notes</dt>
         <dd>
           <div class="input-group">
-            <input type="text" class="form-control" ng-model="outreach.newNote"/>
-            <span class="input-group-btn">
+            <textarea type="text" class="form-control" ng-model="outreach.newNote" cols="30" rows="2" style="resize: vertical;"></textarea>
```

Comment:
That being said, multi line support would be nice. We could look into that for a different change list.